cli/script/command-executor.ts

```diff
@@ -39,16 +39,18 @@ interface NameToCountMap {
     [name: string]: number;
 }
 
+/** Deprecated */
 interface ILegacyLoginConnectionInfo {
     accessKeyName: string;
-    providerName: string;
-    providerUniqueId: string;
-    serverUrl: string;
+    // The 'providerName' property has been obsoleted
+    // The 'providerUniqueId' property has been obsoleted
+    // The 'serverUrl' property has been obsoleted
 }
 
 interface ILoginConnectionInfo {
     accessKey: string;
```

Comment:
I think it's too late, because if we do that then we have the opposite problem where some people still have configuration files with "accessKey" as the property, which we then need to special case for backwards compatibility :P